package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "create-package-json",
-  "version": "0.1.0",
+  "version": "1.0.0-2",
   "description": "Create a package.json",
   "main": "index.js",
   "type": "commonjs",
   "bin": {
-    "create-package-json": "./bin/create-package-json"
+    "create-package-json": "bin/create-package-json"
   },
   "scripts": {
     "test": "standard && mocha",
+    "test:debug": "mocha --inspect --inspect-brk --timeout=0",
```

Comment:
`--timeout=0` should be unnecessary, as Mocha should recognize `--inspect`/`--inspect-brk` and set this automatically

Reply:
I never new this and have it all over my projects!  Is this new? I know I added this in response to tests timing out in while I was at a break point.